ui/src/components/popovers/PasteModal.tsx

```diff
@@ -38,13 +38,12 @@ export default function PasteModal() {
   const setKeyboardLayout = useSettingsStore(
     state => state.setKeyboardLayout,
   );
-
   useEffect(() => {
     send("getKeyboardLayout", {}, resp => {
       if ("error" in resp) return;
       setKeyboardLayout(resp.result as string);
     });
-  }, []);
+  }, [send, setKeyboardLayout]);
```

Comment:
This was a preexisting error in the original code, the React dependencies were simply not included. VS Code pointed the error out :)